### ISSUE TITLE
[A11y] checkbox and radiobuttons - highcontrast and tristate

### ISSUE THREAD

**Issue by osminaz** [lyne-design-system/lyne-components#1602]
- high contrast for checkboxes and radiobuttons should have identical styles.
- tristate in checkbox should have an impact to screenreader. (nice2have)

![image](https://user-images.githubusercontent.com/105784969/217460089-b7a38389-352b-4086-8f1a-1f41b5560db9.png)
![image](https://user-images.githubusercontent.com/105784969/217460135-48e1fb14-1b61-4888-83ba-a7a994d3a020.png)
